OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            
+                        <msItem xml:id="ms_i2.1">
+                            <locus from="7ra" to="7vb"/>
+                            <title ref="LIT3517Epistle" type="incomplete">2 Corinthians 8:1-15</title>
+                            <textLang mainLang="gez"/>
+                            
+                            <incipit xml:lang="gez">
+                                <locus target="#7ra"/>
+                                <hi rend="rubric">ወእነግረክሙ፡ አኃዊነ፡ ጸጋ፡ እግዚአብሔ</hi>ር፡ ዘተውህበ፡ ለቤተ፡ ክርስቲያን፡ ዘመቂዶንያ<hi rend="rubric">፨</hi>
+                                እስመ፡ እምብዙኀ፡ መከራ፡ ሕማሞሙ፡ ፈድፈደ፡ ፍሥሐሆሙ፡ ምስለ፡ ዕመቀ፡ ንዲቶሙ፡ ፈድፈደት፡ ፍሥሐ፡ ብዕሎሙ<hi rend="rubric">፨</hi>
+                                እስመ፡ <gap reason="illegible" unit="chars" extent="1"/>ልየ፡ ሰማዕቶሙ፡ ከመ፡ በአምጣነ፡ ኃይሎሙ<hi rend="rubric">፨</hi>
+                            </incipit>
+                            
+                            <explicit xml:lang="gez">
+                                <locus target="#7vb"/>
+                                እስመ፡ ተረፈ፡ ዚአክሙ፡ ውስተ፡ ንትጋ፡ እልክቱ፡ ይከውን<hi rend="rubric">፨</hi> ወተረፈ፡ ዚአሆሙ፡ ውስተ፡ ንትጋ፡ ዚአክሙ፡ ከመ፡ ይኩን፡ ሀልዎትክሙ፡
+                                ዕሩየ፡ በኵሉ<hi rend="rubric">፨</hi> እስመ፡ ከመዝ፡ ይቤ፡ መጽሐፍ፡ ዘቦ፡ ብዙኃ፡ ኢያትረፈ፡ ወዘቦ፡ ውሑድ፡ ኢያሕፀፀ<hi rend="rubric">፨</hi>
+                            </explicit>
+                        </msItem>
+                        
+                            <msItem xml:id="ms_i2.2">
+                                <locus from="8ra" to="9rb"/>
+                                <title ref="LIT3507Epistle" type="incomplete">1 Peter 4:12-5:10</title>
+                            <textLang mainLang="gez"/>
+                                
+                                <incipit xml:lang="gez">
+                                    <locus target="#8ra"/>
+                                    <hi rend="rubric">አኃዊነ፡ ኢታንክርዋ፡ ለእንተ፡ ትመጽአክሙ፡</hi> መከራ፡ ከመ፡ 
+                                    ዘነኪር፡ ግብር፡ ዘይከውን፡ ብክሙ፡ ዳእሙ፡ መከራ፡ ይእቲ፡ ወዘኢትለምዱ፡ ይበጽሐክሙ፡ ከመ፡ ትሳትፍዎ፡
+                                    በሕማሙ፡ ለክርስቶስ<hi rend="rubric">፨</hi> ወተፈሥሑ፡ ከመ፡ አመ፡ ያስተርኢ፡ በስብሐቲሁ፡
+                                    ትትሐሠዩ፡ ፍሡሐኒክሙ<hi rend="rubric">፨</hi> ወእምሶበ፡ ዘተዓየሩሙ፡ በእንተ፡ ስሙ፡ 
+                                    ለክርስቶስ<hi rend="rubric">፨</hi>
+                                </incipit>
+                                
+                                <explicit xml:lang="gez">
+                                    <locus from="9ra" to="9rb"/>
+                                    ወአጽንዑ፡ ቀዊመ፡ በሃይማኖትክሙ፡ እንዘ፡ ተአምሩ፡ ከመ፡ ሕማሙ፡ ለዝዓለም፡ ትረክቦሙ፡ ለኵሎሙ፡
+                                    አኃዊክሙ፡ አጽንዕዋ፡ ለተፋቅሮ<hi rend="rubric">፤</hi> ወእግዚአብሔር፡ ዘበኵሉ፡ ክብር፡
+                                    ጸውዓክሙ፡ ውስተ፡ ዘለዓለም፡ ስብሐቲሁ፡ በኢየሱስ፡ ክርስቶስ<hi rend="rubric">፨</hi>
+                                    ኅዳጠ፡ ሐሚመ<cb n="b"/>ክሙ፡ ውእቱ፡ ይፈጽም፡ ለክሙ፡ ወያጸንዓክሙ፡ ወያሌብወክሙ፡ ዘሎቱ፡
+                                    ስብሐት፡ ወኃይል፡ ለዓለመ፡ ዓለም፡ አ<hi rend="rubric">፤</hi>
```

Comment:
አ is an abbreviation of አሜን፡, perhaps this might be encoded with `<supplied/>`. What do you think?